From 991ae3dedf978ec24e4355ba96a6d4a6df35bd75 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:57 -0400
Subject: [redhat] kabi: Adding symbol scsi_device_lookup (drivers/scsi/scsi.c)

Message-id: <e8896066819109d9a166c33cba9b9410ac8213e8.1410510773.git.poros@redhat.com>
Patchwork-id: 96058
O-Subject: [PATCH RHEL7.1 177/233] kabi: Adding symbol scsi_device_lookup (drivers/scsi/scsi.c)
Bugzilla: 1071099
RH-Acked-by: Rob Evers <revers@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Find a device given the host. Looks up the scsi_device with
    the specified channel, id, lun for a given host.
    The returned scsi_device has an additional reference that
    needs to be released with scsi_device_put once you're done with it.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/scsi_device_lookup
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/scsi_device_lookup
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/scsi_device_lookup

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/scsi_device_lookup b/redhat/kabi/kabi-module/kabi_ppc64/scsi_device_lookup
new file mode 100644
index 0000000..d3b115f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/scsi_device_lookup
@@ -0,0 +1,2 @@
+#1-
+0x33c80339 scsi_device_lookup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/scsi_device_lookup b/redhat/kabi/kabi-module/kabi_s390x/scsi_device_lookup
new file mode 100644
index 0000000..1d70540
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/scsi_device_lookup
@@ -0,0 +1,2 @@
+#1-
+0x9675e248 scsi_device_lookup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/scsi_device_lookup b/redhat/kabi/kabi-module/kabi_x86_64/scsi_device_lookup
new file mode 100644
index 0000000..e725893
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/scsi_device_lookup
@@ -0,0 +1,2 @@
+#1-
+0x1447ec08 scsi_device_lookup vmlinux EXPORT_SYMBOL
-- 
1.7.1